### PR TITLE
chore: update reset script

### PR DESCRIPTION
**Pull request by rudolfs**
Related to the change we did in: https://github.com/radicle-dev/radicle-upstream/pull/1303.

### CODE REVIEW

File: scripts/reset-state.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@ elif [ "$platform" == "Linux" ]; then
 
     dirs=(
         "$HOME/.radicle"
+        "$HOME/.config/Radicle Upstream"
         "$config_home/radicle"
         "$config_home/radicle-upstream"
```
Comment (xla) [inline scripts/reset-state.sh:23]: This is obsolete now.

**Review by xla — CHANGES_REQUESTED** [review]

File: scripts/reset-state.sh (hunk 1 above)
Comment (rudolfs) [reply scripts/reset-state.sh:23]: Right. Should this perhaps be `"$config_home/Radicle Upstream"` and the one on L21 be deleted?

Comment (xla) [reply scripts/reset-state.sh:23]: yus

Comment (rudolfs) [reply scripts/reset-state.sh]: https://github.com/radicle-dev/radicle-upstream/pull/1325/commits/d641c203dab85cb2d918feba4b5c922e50176ec9

**Review by xla — APPROVED** [review]
🚪 🕴 🕷 🌓

**Review by FintanH — APPROVED** [review]
utACK

**Pull request merged**